squeal-postgresql/src/Squeal/PostgreSQL/Expression/Time.hs

```diff
@@ -156,16 +156,17 @@ type family PGAtTimeZone ty where
       ':<>: 'ShowType pg )
 
 {-|
-Convert a timestamp, timestamp with time zone, or time of day with timezone to a different timezone using an interval offset
+Convert a timestamp, timestamp with time zone, or time of day with timezone to a different timezone using an interval offset or specific timezone denoted by text. When using the interval offset, the interval duration must be less than one day or 24 hours.
 
 >>> printSQL $ (makeTimestamp (2009 :* 7 :* 22 :* 19 :* 45 *: 11.4)) `atTimeZone` (interval_ 8 Hours)
 (make_timestamp((2009 :: int4), (7 :: int4), (22 :: int4), (19 :: int4), (45 :: int4), (11.4 :: float8)) AT TIME ZONE (INTERVAL '8.000 hours'))
 
->>> printSQL $ (makeTimestamptz (2015 :* 9 :* 15 :* 4 :* 45 *: 11.4)) `atTimeZone` (interval_ 1 Days)
-(make_timestamptz((2015 :: int4), (9 :: int4), (15 :: int4), (4 :: int4), (45 :: int4), (11.4 :: float8)) AT TIME ZONE (INTERVAL '1.000 days'))
+>>> printSQL $ (makeTimestamptz (2015 :* 9 :* 15 :* 4 :* 45 *: 11.4)) `atTimeZone` (inline "EST")
```

Comment:
No need for inline generally if OverloadedStrings is on

Reply:
Thanks! I'll remove this and squash the the commits